jobs/doppler/spec

```diff
@@ -126,3 +126,10 @@ properties:
   metron_endpoint.grpc_port:
     description: "The port used to emit grpc messages to the Metron agent"
     default: 3458
+
+  doppler.batch.interval:
+    description: "EXPERIMENTAL: Batching interval for v1 egressing"
+    default: 500000000
```

Comment:
Can we not use the `time.ParseDuration` formats? They are much easier to read and understand.

Reply:
I agree that values like `50ms` are much easier to parse.